components/microsoft_outlook/actions/send-email/send-email.mjs

```diff
@@ -0,0 +1,106 @@
+import microsoft_outlook from "../../microsoft_outlook.app.mjs";
```

Comment:
ESLinst is complaining about name convention here
```suggestion
import microsoftOutlook from "../../microsoft_outlook.app.mjs";
```

Also for this specific action component you can take a look at this api here using the [sdk lib](https://docs.microsoft.com/en-us/graph/api/user-sendmail?view=graph-rest-1.0&tabs=javascript#request)